app/cdap/components/DrawerWidget/IconStore/Underline.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+// This image is an SVG and is not available in MUI Icons Store
+export const Underline = () => {
```

Comment:
Please include a screenshot of the drawer being used

Reply:
added in the screenshots section of the PR description